src/views/FeedList.vue

```diff
@@ -1,7 +1,36 @@
 <template>
   <div class="tips-list">
     <div class="actions">
-      <SendTip />
+      <SendTip :feed="feed" />
```

Comment:
```suggestion
      <SendTip :post="feed == 'posts'" />
```
Proposing to don't parse `feed` internally and to switch the mode from here